src/pybind/mgr/dashboard/rgw_detail.html

```diff
@@ -0,0 +1,101 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+<script>
+        $(document).ready(function(){
+            // Pre-populated initial data at page load
+            var content_data = {{ content_data }};
+
+            var refresh = function() {
+                $.get("{{ url_prefix }}/rgw/rgw_data/" + content_data.rgw_id + "/", function(data) {
+                   content_data = data;
+                   setTimeout(refresh, 3000);
+                });
+            };
+            setTimeout(refresh, 5000);
+
+            var post_load = function() {
+                content_data.rgw_metadata_list = [];
+                content_data.rgw_status_list = [];
+                _.each(content_data.rgw_metadata, function(v, k) {
+                    content_data.rgw_metadata_list.push({
+                        key: k,
+                        value: v
+                    });
+                });
+                _.each(content_data.rgw_status, function(v, k) {
+                    content_data.rgw_status_list.push({
+                        key: k,
+                        value: v
+                    });
+                });
+                
+                if (content_data.rgw_status_list.length() == 0) {
+                     content_data.rgw_status_show = false;
+                } else {
+                     content_data.rgw_status_show = true;
+                }
+            };
+
+            rivets.bind($("#content"), content_data);
+            post_load();
+        });
+</script>
+
+<section class="content-header">
+    <h1>
+        rgw.{rgw_id}
+    </h1>
+</section>
+
```

Comment:
Start a `section class="content"` here, to get proper padding/margins on the main part with the tables.